__init__.py

```diff
@@ -44,8 +45,6 @@
             print("The Retriever lacks permission to access the ~/.retriever/ directory.")
             raise
 SCRIPT_SEARCH_PATHS = [
```

Comment:
This script search path needs to be defined.
@ethanwhite, could you help us confirm the path for which scripts should be searched?